src/k8s/production.yml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: marketplace-rest-api
+  namespace: webapps
+spec:
+  selector:
+    matchLabels:
+      app: marketplace-rest-api
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: marketplace-rest-api
+        environment: production
+    spec:
+      containers:
+      - name: app
+        image: eclipsefdn/marketplace-rest-api:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+        resources:
+          limits:
```

Comment:
@autumnfound any recommendation on limits?

Reply:
I think having 2 cores + 2gb would be a good start. It would give the cache some breathing room, along with the response marshalling :)

Reply:
Thx for the recommendations. I made the changes. I will now add @mbarbero to review this PR.